types/storybook__addon-info/index.d.ts

```diff
@@ -1,50 +1,68 @@
-// Type definitions for @storybook/addon-info 4.1
+// Type definitions for @storybook/addon-info 5.2
 // Project: https://github.com/storybookjs/storybook, https://github.com/storybookjs/storybook/tree/master/addons/info
 // Definitions by: Mark Kornblum <https://github.com/mkornblum>
 //                 Mattias Wikstrom <https://github.com/fyrkant>
 //                 Kevin Lee <https://github.com/RunningCoderLee>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 3.1
 
-import * as React from 'react';
-import { RenderFunction, StoryDecorator } from '@storybook/react';
+import { ComponentType, ReactElement } from 'react';
+import { DecoratorFunction, StoryFn, StoryContext, Parameters, StoryApi } from '@storybook/addons';
 
 export interface WrapStoryProps {
-  storyFn?: RenderFunction;
-  context?: object;
-  options?: object;
+    storyFn?: StoryFn;
+    context?: object;
+    options?: object;
 }
 
 export interface Options {
-  text?: string;
-  header?: boolean;
-  inline?: boolean;
-  source?: boolean;
-  propTables?: Array<React.ComponentType<any>> | false;
-  propTablesExclude?: Array<React.ComponentType<any>>;
-  styles?: object;
-  components?: { [key: string]: React.ComponentType<any> };
-  marksyConf?: object;
-  maxPropsIntoLine?: number;
-  maxPropObjectKeys?: number;
-  maxPropArrayLength?: number;
-  maxPropStringLength?: number;
-  TableComponent?: React.ComponentType<{
-    propDefinitions: Array<{
-      property: string;
-      propType: object | string; // TODO: info about what this object is...
-      required: boolean;
-      description: string;
-      defaultValue: any;
-    }>
-  }>;
-  excludedPropTypes?: string[];
+    text?: string;
+    header?: boolean;
+    inline?: boolean;
+    source?: boolean;
+    propTables?: Array<ComponentType<any>> | false;
+    propTablesExclude?: Array<ComponentType<any>>;
```

Comment:
Use `unknown` instead? 🤔

```suggestion
    propTablesExclude?: Array<ComponentType<unknown>>;
```

Or just delete the `any`? 🤔
```suggestion
    propTablesExclude?: ComponentType[];
```